environments/minikube.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I don't really think a fork is a place to be putting opinionated deployment configuration. We should maintain this as a way to track changes from the upstream code base. Our configuration and deployment tools should live elsewhere.

Reply:
Then it sounds like we should also have a private repository of this fork to maintain custom changes: e.g. deployment configurations.